frontend/src/components/pages/teacher/DisplayAssessmentResultsPage/DisplayAssessmentResultsByStudentPage.tsx

```diff
@@ -0,0 +1,215 @@
+  {
+    id: "8",
+    firstName: "John",
+    lastName: "Doe",
+    studentNumber: "245",
+    isViewed: true,
+  },
+  {
+    id: "9",
+    firstName: "Jane",
+    lastName: "Doe",
+    studentNumber: "246",
+    isViewed: false,
+  },
+  {
+    id: "10",
+    firstName: "Z",
+    lastName: "Z",
+    studentNumber: "247",
+    isViewed: true,
+  },
+  {
+    id: "11",
+    firstName: "Z",
+    lastName: "Z",
+    studentNumber: "248",
+    isViewed: false,
+  },
+  {
+    id: "12",
+    firstName: "Z",
+    lastName: "Z",
+    studentNumber: "249",
+    isViewed: true,
+  },
+  {
+    id: "13",
+    firstName: "Z",
+    lastName: "Z",
+    studentNumber: "250",
+    isViewed: false,
+  },
+  {
+    id: "14",
+    firstName: "Z",
+    lastName: "Z",
+    studentNumber: "251",
+    isViewed: true,
+  },
+  {
+    id: "15",
+    firstName: "Z",
+    lastName: "Z",
+    studentNumber: "252",
+    isViewed: false,
+  },
+  {
+    id: "16",
+    firstName: "Z",
+    lastName: "Z",
+    studentNumber: "253",
+    isViewed: true,
+  },
+  {
+    id: "17",
+    firstName: "Z",
+    lastName: "Z",
+    studentNumber: "254",
+    isViewed: false,
+  },
+  {
+    id: "18",
+    firstName: "Z",
+    lastName: "Z",
+    studentNumber: "255",
+    isViewed: true,
+  },
+  {
+    id: "19",
+    firstName: "Z",
+    lastName: "Z",
+    studentNumber: "256",
+    isViewed: false,
+  },
+  {
+    id: "20",
+    firstName: "Z",
+    lastName: "Z",
+    studentNumber: "257",
+    isViewed: true,
+  },
+  {
+    id: "21",
+    firstName: "Z",
+    lastName: "Z",
+    studentNumber: "258",
+    isViewed: false,
+  },
+];
+
+const STUDENT_STATISTICS_CONFIG: StatisticsConfig = {
+  totalScore: {
+    title: "total score",
+    formatValue: PERCENTAGE_FORMAT,
+  },
+  percentile: {
+    title: "percentile",
+    formatValue: NTH_FORMAT,
+  },
+};
+
+const DisplayAssessmentResultsByStudentPage = () => {
+  const [selectedStudentId, setSelectedStudentId] = React.useState<string>();
+
+  const students = MOCK_STUDENTS;
+  const idToStudentMap = React.useMemo(
+    () =>
+      students.reduce((acc, student) => {
+        acc[student.id] = student;
+        return acc;
+      }, {} as Record<string, (typeof students)[number]>),
+    [students],
+  );
+  const currentStudent = idToStudentMap[selectedStudentId || ""];
+
+  return (
+    <Flex gap={14} h="calc(100vh - 350px)">
+      <StudentList
+        selectedStudentId={selectedStudentId}
+        setSelectedStudentId={setSelectedStudentId}
+        students={students}
+      />
+      <Flex align="start" direction="column" flex={1} gap={10}>
+        {selectedStudentId ? (
+          <>
+            <Text color="blue.300" textStyle="subtitle1">
+              {`${currentStudent.lastName}, ${currentStudent.firstName}`}
+            </Text>
+            <StatisticsSection
+              config={STUDENT_STATISTICS_CONFIG}
+              values={{ totalScore: 50, percentile: 10 }}
+            />
+            <Divider />
```

Comment:
nit: can we colour the divider according to the design? i think it should be a tad darker

Reply:
is this something that should be done in general? like a theme? or are these dividers specifically darker

Reply:
i think all the dividers are actually `grey.200`, but we've been setting the color prop manually. we can probably set to theme & then get rid of that prop from all the dividers!